coding_schemes/Cash_Modality_Yesno.json

```diff
@@ -0,0 +1,174 @@
+{
+  "Codes": [
+    {
+      "CodeID": "code-5740cafe5c18",
+      "CodeType": "Normal",
+      "DisplayText": "Prefer current modality",
+      "NumericValue": 0,
+      "StringValue": "prefer_current_modality",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-5f7c2445b240",
+      "CodeType": "Normal",
+      "DisplayText": "Safety ",
```

Comment:
any reason for the space here?

Reply:
This is because of csv, they're generated from.